### DISCUSSION TITLE
Join and count()

### DISCUSSION THREAD

**Discussion by digi604** [supabase/supabase#13532] [Questions]
I have 2 tables:

plugins with id, name
plugin_vote with id, plugin_id and user_id

i want one query with supabase js that returns me all plugins with their votes as numbers.

can someone help me? GPT4 couldnt answer

**Comment by GaryAustin1** [comment]
![image](https://user-images.githubusercontent.com/54564956/230731954-58e199be-25d2-42e6-ad03-9e3edd9519cd.png)
